src/pveclib/vec_int16_ppc.h

```diff
@@ -482,6 +482,13 @@ __test_div10000 (vui16_t n)
  */
 
 ///@cond INTERNAL
+#ifndef vec_popcnth
+static inline vui16_t vec_popcnth (vui16_t vra);
+#else
+/* Work around for GCC PR85830.  */
+#undef vec_popcnth
+#define vec_popcnth __builtin_vec_vpopcnth
+#endif
```

Comment:
same comments as for analogous code, above

Reply:
See response above.